vignettes/geologic-map.Rmd

```diff
@@ -39,46 +39,46 @@ install.packages("devtools")
 devtools::install_github("palaeoverse/rmacrostrat")
 ```
 
-You can now load `rmacrostrat` using the default `library` function:
+You can now load `rmacrostrat` using the standard `library` function:
 
 ```{r load-library, eval = TRUE}
 library(rmacrostrat)
 ```
 
-**Before we get onto the good stuff, the development team has a small request**. If you use `rmacrostrat` in your research, please cite the associated publication. This will help us to continue our work in supporting you to do yours. You can access the appropriate citation via:
+**Before we get into the good stuff, the development team has a small request**. If you use `rmacrostrat` in your research, please cite the associated publication. This will help us to continue our work in supporting you to do yours. You can access the appropriate citation via:
 
 ```{r eval = TRUE}
 citation("rmacrostrat")
 ```
 
 # Context
-The Hell Creek is a geologic formation from the late Cretaceous and early Paleogene of North America, which is  found cropping out across Montana, North and South Dakota, and Wyoming in the United States. It consists of both freshwater and brackish clays, mudstones, and sandstones, representing a fluvial system on the shoreline of the Western Interior Seaway. The Hell Creek is famous for its preservation of an extensive array of both terrestrial and marginal marine fossils from just prior to the K/Pg mass extinction, including widely recognisable dinosaurs such as *Edmontosaurus*, *Triceratops* and *Tyrannosaurus rex*. In this vignette, we will show you how to use `rmacrostrat` to find the outcrop of the Hell Creek Formation, and use this to create a map which could be used in a potential publication. Whilst we have chosen a specific formation, this methodology could be applied to any number of geologic units of interest. So let's dive in!
+The Hell Creek is a geologic formation from the late Cretaceous and early Paleogene of North America, which is found cropping out across Montana and North and South Dakota, in the United States. It consists of both freshwater and brackish clays, mudstones, and sandstones, deposited by a fluvial system on the shoreline of the Western Interior Seaway. The Hell Creek is famous for its preservation of an extensive array of both terrestrial and marginal marine fossils from just prior to the K/Pg mass extinction (66 million years ago), including widely recognizable dinosaurs such as *Edmontosaurus*, *Triceratops* and *Tyrannosaurus rex*. In this vignette, we will show you how to use `rmacrostrat` to find the outcrop of the Hell Creek Formation, and use this to create a map which could be used in a potential publication. Whilst we have chosen a specific formation, this approach could be applied to any number of geologic units of interest. So, let's dive in!
```

Comment:
'found cropping out' feels like a weird phrase but I guess it fits? :)

Reply:
It sounds odd, but it's the correct geologic(al? lol) term; an outcrop is rocks at the surface, 'cropping out' is those rocks appearing at the surface, if that makes sense.